dist/src/main/resources/application-opensearch.yaml

```diff
@@ -122,6 +122,13 @@ zeebe:
             maxCacheSize: 10000
 
 camunda:
+
+  # Unified Configuration (TODO: cleanup later)
+  data:
+    secondary-storage:
+      type: 'opensearch'
+  # -----
```

Comment:
:wrench: Same here. Please add the new properties and remove the legacy ones.

Reply:
Removed